firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -56,11 +62,16 @@ def test_notification_bar(self):
 
                 # Navigate to the unsafe page
                 # Check "ignore warning" link then notification bar's "not badware" button
-                self.marionette.navigate(unsafe_page)
-                # Wait for the DOM to receive events for about:blocked
-                time.sleep(1)
-                self.check_ignore_warning_button(unsafe_page)
-                self.check_not_badware_button(label, report_page)
+                # Only do this if feature supports it
+                if label is not None:
+                    self.marionette.navigate(unsafe_page)
+                    # Wait for the DOM to receive events for about:blocked
+                    time.sleep(1)
+                    self.check_ignore_warning_button(unsafe_page)
```

Comment:
The ignore warning link is still present for the unwanted software case. So you should not exclude that from testing. It's really only the not badware link which you cannot test for that page type.